src/Markdig/Extensions/Yaml/YamlFrontMatterParser.cs

```diff
@@ -1,44 +1,128 @@
 // Copyright (c) Alexandre Mutel. All rights reserved.
 // This file is licensed under the BSD-Clause 2 license. 
 // See the license.txt file in the project root for more information.
+using Markdig.Helpers;
 using Markdig.Parsers;
+using Markdig.Syntax;
 
 namespace Markdig.Extensions.Yaml
 {
     /// <summary>
     /// Block parser for a YAML frontmatter. 
     /// </summary>
-    /// <seealso cref="Markdig.Parsers.FencedBlockParserBase{YamlFrontMatterBlock}" />
-    public class YamlFrontMatterParser : FencedBlockParserBase<YamlFrontMatterBlock>
+    /// <seealso cref="YamlFrontMatterBlock" />
+    public class YamlFrontMatterParser : BlockParser
     {
-        // We reuse a FencedCodeBlock parser to grab a frontmatter, only active if it happens on the first line of the document.
+		// We reuse a FencedCodeBlock parser to grab a frontmatter, only active if it happens on the first line of the document.
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="FencedCodeBlockParser"/> class.
-        /// </summary>
-        public YamlFrontMatterParser()
+		/// <summary>
```

Comment:
spaces instead of tabs for indentation for the whole file

Reply:
Corrected. Thanks. Have you considered adding an [.editorconfig](http://editorconfig.org/) to the root of the project to help make sure IDE's and tools are aware of the preference?

Reply:
sure I will